client/src/components/HistoryViewer/HistoryViewer.js

```diff
@@ -246,21 +251,43 @@ class HistoryViewer extends Component {
     );
   }
 
+  renderComparisonSelectionList() {
+    const { compare: { versionFrom }, ListComponent } = this.props;
+    if (!versionFrom) {
+      return null;
+    }
+    return (
+      <ListComponent
+        versions={[versionFrom]}
+        extraClass="history-viewer__table history-viewer__table--comparison-selected"
+      />
+    );
```

Comment:
Do you think it should be the responsibility of HistoryViewer to do this? I feel a little bit like it should be the `HistoryViewerVersionList` component that does it